README.rst

```diff
@@ -34,6 +34,61 @@ One Time Setup
   # Here's how you might do that if you have virtualenvwrapper setup.
   mkvirtualenv -p python3.8 learning-assistant
 
+In your ``private.txt`` requirements file in edx-platform, add:
+
+.. code-block::
+
+  -e /edx/src/learning-assistant
+
+In your ``private.py`` settings file in edx-platform, add the below settings. The value of the API key shouldn't matter, because it's not being used at this point, but the setting needs to be there.
```

Comment:
Can you specify `lms/envs/private.py`, please? The `private.py` file does not come with the repository, so a user has to add it themselves. If they don't already have this file, it may be unclear where the file should be.